modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -115,6 +118,11 @@
     /** */
     private static final String METASTORE_CURR_BLT_KEY = "metastoreBltKey";
 
+    /** Warning of unsafe deactivation. */
+    public static final String DATA_LOST_ON_DEACTIVATION_WARNING =
+        "Cluster has caches configured without persistence. " +
+            "During deactivation in-memory data and objects can be lost!";
```

Comment:
We should change *can* to *will* because there is no case whe in-memory data will not be lost after deactivation.
So correct message to the user

`Deactivation stopped. Cluster has in-memory caches(without persistence). During deactivation, in-memory data will be lost.'

Reply:
Fixed.